pkg/vault/test/authorizer/TimelockAuthorizerExecute.test.ts

```diff
@@ -165,6 +165,13 @@ describe('TimelockAuthorizer execute', () => {
       }
     });
 
+    it('emits ExecutionScheduled event', async () => {
+      const receipt = await authorizer.instance.connect(user).schedule(authenticatedContract.address, data, []);
+
+      // _scheduledExecutions doesn't have a getter so we can't fetch it before the call
+      expectEvent.inReceipt(await receipt.wait(), 'ExecutionScheduled', { scheduledExecutionId: 2 });
```

Comment:
I don't understand this comment, what do you meant?

Also, let's test the `actionId` param of the event.